tests/reconfiguration.py

```diff
@@ -440,13 +440,6 @@ def run(args):
             test_add_node_from_snapshot(network, args)
             test_add_node_from_snapshot(network, args, from_backup=True)
             test_add_node_from_snapshot(network, args, copy_ledger_read_only=False)
```

Comment:
We can now start a node from a snapshot without any ledger at all, whereas previously, a node would automatically reject snapshots whose evidence aren't covered by the ledger, are replay the entire ledger history instead.